src/tests/class_tests/openms/source/FineIsotopeDistribution_test.cpp

```diff
@@ -77,7 +77,7 @@ START_SECTION(( IsotopeDistribution run(const EmpiricalFormula&) const ))
 
   // simple way of getting an IsotopeDistribution using total probability
   test_id = ef.getIsotopeDistribution(FineIsotopePatternGenerator(0.01, true, false));
-  TEST_EQUAL(test_id.size(), 6)
+  TEST_EQUAL(test_id.size(), 3)
```

Comment:
why now 3?

Reply:
Again, because we're doing p_trimming by default